tasks/packages.yml

```diff
@@ -5,12 +5,6 @@
     update_cache: yes
   when: ansible_pkg_mgr == "apt"
 
-- name: update the yum package index
```

Comment:
I'd rather use `update_cache` option as mentioned in #3 instead of removing this.

Reply:
ok, but then we would need to make two tasks, one for apt, one for yum. `update_cache` only works together when state is `present `or `latest` and the generic package module does not have a update_cache option
Did it in michaelsstuff/ansible-role-steamcmd@5e10d8d